hack/install-router.sh

```diff
@@ -13,8 +13,8 @@ OPENSHIFT="${3}"
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 if [[ "${OPENSHIFT}" == "" ]]; then
-    if [[ "$(which osc)" != "" ]]; then
-        OPENSHIFT=$(which osc)
+    if [[ "$(which openshift)" != "" ]]; then
```

Comment:
Unless you have a need to call the openshift command (which i can't think why), use osc.

----- Original Message -----
> > @@ -13,8 +13,8 @@ OPENSHIFT="${3}"
> >  OS_ROOT=$(dirname "${BASH_SOURCE}")/..
> >  
> >  if [[ "${OPENSHIFT}" == "" ]]; then
> > -    if [[ "$(which osc)" != "" ]]; then
> > -        OPENSHIFT=$(which osc)
> > +    if [[ "$(which openshift)" != "" ]]; then
> 
> It's worth discussing whether we should be calling 'openshift cli' or 'osc'.
> The build will produce the symlinks for 'osc' now.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/652/files#r23390335